README.md

```diff
@@ -7,6 +7,106 @@
 
 This is a desktop application acting as a Gradle Tooling API client.
 
+## Usage
+
+You can download the latest version of _Gradle Client_ for macos, Linux or Windows from the [Releases](https://github.com/gradle/gradle-client/releases) assets.
```

Comment:
💭 There aren't any releases here yet.

Reply:
Right. I just published the first release, PTAL.